Python/Product/Analysis/AnalysisUnit.cs

```diff
@@ -346,13 +346,9 @@ internal AnalysisUnit(ScopeStatement ast, InterpreterScope scope)
 
         public LocationInfo ResolveLocation(object location) {
             Node node = (Node)location;
-            MemberExpression me = node as MemberExpression;
-            SourceSpan span;
-            if (me != null) {
-                span = me.GetNameSpan(Tree);
-            } else {
-                span = node.GetSpan(Tree);
-            }
+            var span = (node as MemberExpression)?.GetNameSpan(Tree)
+                ?? (node as Parameter)?.NameExpression?.GetSpan(Tree)
+                ?? node.GetSpan(Tree);
```

Comment:
Is it possible that `Parameter.NameExpression` is `null`?

Reply:
Very theoretically. No harm in making sure here though.

Reply:
I'm asking because semantics depends on it. There is a difference between

- If node is `Parameter`, then call `Parameter.NameExpression.GetSpan`
- If it is not, call `node.GetSpan` 

and

 - If node is `Parameter` and `NameExpression != null` then call `GetSpan`, otherwise fallback to `node.GetSpan(Tree)`